msal/application.py

```diff
@@ -917,6 +1005,11 @@ def acquire_token_on_behalf_of(self, user_assertion, scopes, **kwargs):
 
         :param str user_assertion: The incoming token already received by this app
         :param list[str] scopes: Scopes required by downstream API (a resource).
+        :param claims_challenge:
+            The claims_challenge parameter requests specific claims requested by the resource provider
+            in the form of a claims_challenge directive in the www-authenticate header to be
+            returned from the UserInfo Endpoint and/or in the ID Token and/or Access Token.
+            It is a string of a JSON object which contains lists of claims being requested from these locations..
```

Comment:
Duplicate periods (.).

Reply:
@abhidnya13 Jiashuo has sharp eyes... :-)

Reply:
He does :-) Jiashuo,  I have created a [new PR](https://github.com/AzureAD/microsoft-authentication-library-for-python/pull/250/files) addressing this and the other concern you have, please take a look :-) For some reason, I cannot add you as a reviewer.